README.md

```diff
@@ -1,3 +1,13 @@
+### Build Status
```

Comment:
I installed Codecov in this repository, so the codecov.io links below can be updated to:
* https://codecov.io/github/tulip-control/floras/graph/badge.svg
* https://app.codecov.io/github/tulip-control/floras

I am not 100% confident the first one will work, but it follows the pattern...

The other URLs to GitHub Workflow badges should be updated to tulip-control/floras instead of your fork (tulip-control/floras).

Reply:
The Codecov currently shows as "unknown", but it should show a status after this PR is merged.